addons/web_tour/static/src/js/running_tour_action_helper.js

```diff
@@ -95,14 +95,19 @@ var RunningTourActionHelper = core.Class.extend({
                 bubbles: true,
             }));
         } else if (values.$element.is("select")) {
-            var $options = values.$element.children("option");
+            var $options = values.$element.find("option");
             $options.prop("selected", false).removeProp("selected");
             var $selectedOption = $options.filter(function () { return $(this).val() === text; });
             if ($selectedOption.length === 0) {
                 $selectedOption = $options.filter(function () { return $(this).text().trim() === text; });
             }
+            if ($selectedOption.length === 0 && text.startsWith('index ')) {
+                const index = parseInt(text.substring(6));
```

Comment:
Will need something more robust here, what if you want the 11th one?
Either a regex or assuming nothing else should be after `index ` and just take everything, maybe even split on `index `

Reply:
Rewritten using a regex